src/main/java/com/cleanroommc/groovyscript/compat/mods/betterwithmods/HopperFilters.java

```diff
@@ -125,6 +126,10 @@ public void validate(GroovyLog.Msg msg) {
             validateItems(msg, 0, Integer.MAX_VALUE, 0, 0);
             validateFluids(msg);
             msg.add(IngredientHelper.isEmpty(filter), "filter must be defined");
+            // Ignores stack size
+            if (IngredientHelper.overMaxSize(filter, 1)) {
+                msg.add("Filter must have stack size of 1, got {}", filter.getAmount());
+            }
```

Comment:
```suggestion
            msg.add(IngredientHelper.overMaxSize(filter, 1), "Filter must have stack size of 1, got {}", filter.getAmount());
```

Reply:
Yeah this was a leftover from when the condition was very wordy.